cmd/template_delete.go

```diff
@@ -0,0 +1,65 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/exoscale/egoscale"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	templateCmd.AddCommand(templateShowCmd)
+}
+
+// templateDeleteCmd represents the delete command
+var templateDeleteCmd = &cobra.Command{
+	Use:   "delete <template id>+",
+	Short: "Delete a template",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) < 1 {
+			return cmd.Usage()
+		}
+
+		force, err := cmd.Flags().GetBool("force")
+		if err != nil {
+			return err
+		}
+
+		tasks := make([]task, 0, len(args))
+		for _, arg := range args {
+			id, err := egoscale.ParseUUID(arg)
+			if err != nil {
+				return err
+			}
+			cmd := egoscale.DeleteTemplate{
+				ID: id,
+			}
+			if err != nil {
+				return err
+			}
+
+			if !force {
+				if !askQuestion(fmt.Sprintf("Are you sure you want to delete template: %q", arg)) {
```

Comment:
If you actually ask the user a question the message format should be:

```suggestion
				if !askQuestion(fmt.Sprintf("Are you sure you want to delete template %q?", arg)) {
```

Reply:
Updated, thanks.